src/OrbitGl/ThreadTrack.cpp

```diff
@@ -486,7 +486,7 @@ void ThreadTrack::UpdatePrimitives(Batcher* batcher, uint64_t min_tick, uint64_t
       const Vec2& size = text_box.GetSize();
 
       if (text_box.Duration() > draw_data.ns_per_pixel) {
-        if (collapse_toggle_->IsExpanded()) {
+        if (!collapse_toggle_->IsCollapsed()) {
```

Comment:
Should we (in a future PR) maybe change this behavior? I think it's reasonable to expect that `IsCollapsed() != IsExpanded` holds... we could only provide one of both methods, and `IsCollapsible()` in addition to deal with single-level tracks, WDYT? Could you create a bug for this?

Reply:
Reading your initial commit, maybe the solution should rather be to have this a tri-state Collapsed / Expanded / Inactive.

Reply:
>Reading your initial commit, maybe the solution should rather be to have this a tri-state Collapsed / Expanded / Inactive.

This is how it is currently implemented. There is this tri-state and there are methods for IsCollapsed, IsExpanded, IsInactive. I simply was not aware of the third state. Not sure about a nice solution here. It is probably not too bad as it is now.